src/js/components/Layout/Breadcrumbs.jsx

```diff
@@ -1,11 +1,67 @@
-import React from 'react';
+/* eslint-disable react/no-array-index-key */
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { showModal } from '../../actions';
 
-const Breadcrumbs = () => (
-  <div className="bradcrumbs-container d-flex">
-    <span>
-      Breadcrumbs
-    </span>
-  </div>
-);
+class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
 
-export default Breadcrumbs;
+    this.openModal = this.openModal.bind(this);
+  }
+
+  openModal() {
+    this.props.showModal();
+  }
+
+  render() {
+    const listItems = window.location.pathname.split('/');
```

Comment:
@julienreliefapps Yeah I was really hoping for the breadcrumbs to be pulled from the API in order to handle more complex requirements like translations on the backend. But if you can make this work with translations on the client-side then be my guest.

Reply:
Ah my bad I really didn't notice about the translation, and @jmiranda in this case, it's true that it may be easier pulling it from the backend

Reply:
It actually might even be just as easy to render the breadcrumbs as a global javascript variable on the page since I'm realizing that getting it from the API might be tough to do. For example, in order for an endpoint to resolve the breadcrumbs you'll need to pass the full URL, which isn't terrible. Just seems odd. 

Alternatively, maybe the page itself (_react.gsp) can render a list of breadcrumbs as a global variable?

@awalkowiak Thoughts?

Reply:
@jmiranda  should I use the translationService ?